ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/admin/ledger_pruning_service.proto

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.digitalasset.ledger.api.v1.admin;
+
+import "google/protobuf/timestamp.proto";
+
+option java_outer_classname = "LedgerPruningServiceOuterClass";
+option java_package = "com.digitalasset.ledger.api.v1.admin";
+option csharp_namespace = "Com.DigitalAsset.Ledger.Api.V1.Admin";
+
+// Status: experimental interface, will change before it is deemed production
+// ready
+
+// Prunes/truncates the "oldest" ledger transactions from the ledger (specifically the Ledger Api Server) by removing
```

Comment:
Isn't this service supposed to be about pruning the participant and not the underlying ledger (which of course could be distributed and not prunable from a single point). I would reword the comments to talk explicitly about the participant, though we'll likely need to explain here what this would mean in practice (e.g. the index is pruned, and possible some participant-local storage).